app/controllers/admin/news_controller.rb

```diff
@@ -63,8 +63,31 @@ def destroy
   protected
 
   def permitted_parameters
-    params.require(:blog).permit(*%w(title body user_id published_at post_date post_now tags published old_title_slug
-                                     timezone description_abbr update_title is_listicle listicles_attributes user_email index_image_id index_image).map(&:to_sym).freeze)
+    params.require(:blog).permit(
+      :body,
+      :description_abbr,
+      :index_image,
+      :index_image_id,
+      :is_listicle,
+      :language,
+      :listicles_attributes,
+      :old_title_slug,
+      :post_date,
+      :post_now,
+      :published,
+      :published_at,
+      :tags,
+      :timezone,
+      :title,
+      :update_title,
+      :user_email,
+      :user_id,
+    )
+  end
+
+  def normalize_params
+    language = params.dig(:blog, :language)
```

Comment:
When would there not be a language parameter in the update? Why not just include this in the parameters?

Reply:
> When would there not be a language parameter in the update? 

I can't think of a situation where that would be the case.

> Why not just include this in the parameters?

It's ambiguous what "this" refers to here — orphaned pronouns are like missing external dependencies. 


If you mean why are we casting the language param to an integer here, it's because ActiveRecord raises an exception on `"1"` when it needs `1` 🤷‍♂ 

```
Blog.where(language: "1").first.update(language: "1")

Blog Load (0.6ms)  SELECT  "blogs".* FROM "blogs" WHERE "blogs"."language" = $1  ORDER BY published_at desc LIMIT 1  [["language", 1]]
 (0.1ms)  BEGIN
 (0.1ms)  ROLLBACK
ArgumentError: '1' is not a valid language
```

Reply:
Awwwww bummers. I think this might be fixed in Rails 5.

Ok